src/style.css

```diff
@@ -0,0 +1,54 @@
+html {
+  box-sizing: border-box;
+}
```

Comment:
This will works only for html tag, (this rule is not inherited by default) if you want to set it for every element use * selector